src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/excel/MultipleSingleMidEventValidationService.java

```diff
@@ -137,7 +139,11 @@ private void populateDynamicLists(String userToken, String caseTypeId,
 
         }
 
-        multipleData.setBatchUpdateJurisdiction(populateDynamicList(jurCodesCollection));
+        DynamicListType dynamicList = new DynamicListType();
+        dynamicList.setDynamicList(new DynamicFixedListType());
+        dynamicList.getDynamicList().setListItems(jurCodesCollection);
+
+        multipleData.setBatchUpdateJurisdictionList(ListTypeItem.from(dynamicList));
```

Comment:
potential issue if jurCodesCollection is empty. if no codes exists, dynamic list will be empty but definition files have it as a mandatory field. if no options are available then user would be unable to submit

Reply:
Good catch, fixed